vcd/resource_vcd_org_vdc_access_control_test.go

```diff
@@ -0,0 +1,189 @@
+//go:build vdc || ALL || functional
+// +build vdc ALL functional
+
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"strings"
+	"testing"
+)
+
+func TestAccVcdOrgVdcAccessControl(t *testing.T) {
+	preTestChecks(t)
+	if !usingSysAdmin() {
+		t.Skip(t.Name() + " requires system admin privileges")
+	}
+
+	orgUserPasswordFile := "org_user_pwd.txt"
+	userName1 := strings.ToLower(t.Name())
+	userName2 := strings.ToLower(t.Name()) + "2"
+	accessControlName := "test-access-control"
```

Comment:
better to use real test name :)

Reply:
If you're speaking about lower case the test name, users do not contain uppercase characters. If you create a user with uppercase characters, VCD lowercase them.

Reply:
anything which allows in UI to see that's TestAccVcdOrgVdcAccessControl who left resource (if something went wrong). :)

Reply:
In tests the only things created are:
2 users, which are using the test name as user name
1 VDC that is being created to test `vdc_id` which its name is the test name.

No more actual resources are being created at VCD side.

`test-access-control` is only being used as resource name, and in the UI access control doesn't have a name by itself.